etc/nextcloudpi-config.d/nc-swapfile.sh

```diff
@@ -26,8 +26,27 @@ is_active()
 
 configure()
 {
+  cat  > /etc/systemd/system/ncp-swapfile.service <<'EOF'
+
+[Unit]
+Description= ncp-swapfile service
+
+[Service]
+Type=simple
+ExecStart=/sbin/dphys-swapfile setup
+ExecStartPost=/sbin/dphys-swapfile swapon
+ExecStop=/sbin/dphys-swapfile swapoff
+
+[Install]
+Requires=nc-automount.services
+After=nc-automount.service
+
+EOF
+
+  systemctl daemon-reload
+
   local ORIG=$( grep -oP "CONF_SWAPFILE=.*" /etc/dphys-swapfile | cut -f2 -d= )
-  [[ "$ORIG" == "$SWAPFILE_" ]] && return
+  [[ "$ORIG" -ef "$SWAPFILE_" ]] && return
```

Comment:
I just tested this three times and it does not fail for me with the original ==, but I still think that new way is better, let's keep it